examples/viewport-animation/src/app.js

```diff
@@ -39,34 +31,25 @@ export default class App extends Component {
     window.removeEventListener('resize', this._resize);
   }
 
-  _resize = () => {
-    this.setState({
-      viewport: {
-        ...this.state.viewport,
-        width: this.props.width || window.innerWidth,
-        height: this.props.height || window.innerHeight
-      }
-    });
-  };
-
-  _easeTo = ({longitude, latitude}) => {
-    // Remove existing animations
-    TWEEN.removeAll();
+  _onViewportChange = viewport => this.setState({
+    viewport: {...this.state.viewport, ...viewport}
+  });
 
-    const {viewport} = this.state;
+  _resize = () => this._onViewportChange({
+    width: this.props.width || window.innerWidth,
+    height: this.props.height || window.innerHeight
```

Comment:
This might trigger a `transition`, we should overwrite 'transitionDuration' to 0 here. (Might need same change in `viewport-animation` example of deck.gl but should be handled correctly in other examples. Try re-sizing the window.

Reply:
Good point.
Now I think about it, should we remove width and height from interpolation? And automatically kill the transition in transition manager if the canvas size changed?

Reply:
We don't interpolate width and height.

Reply:
As discussed, I removed `width` and `height` from viewport comparison.